plugins/poll/plugin.rb

```diff
@@ -398,6 +398,18 @@ def vote
       end
     end
 
+    def remove_vote
+      post_id   = params.require(:post_id)
+      poll_name = params.require(:poll_name)
+
+      begin
+        poll, options = DiscoursePoll::Poll.vote(post_id, poll_name, [], current_user, remove: true)
+        render json: { poll: poll, vote: options }
+      rescue StandardError => e
```

Comment:
This is not related to your change but I wished we had use a custom error class instead. This is rescuing all errors whether they are related to poll vote or not.